migration/versions/bbd3bba567c1_banners.py

```diff
@@ -0,0 +1,30 @@
+"""Banners
+
+Revision ID: bbd3bba567c1
+Revises: 6ba4018c2666
+Create Date: 2017-12-16 02:51:54.685902
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'bbd3bba567c1'
+down_revision = '6ba4018c2666'
+
+from alembic import op
+import sqlalchemy as sa
+from depot.fields.sqlalchemy import UploadedFileField
+from depot.fields.specialized.image import UploadedImageWithThumb
+
+
+def upgrade():
+    op.create_table(
+        'banner',
+        sa.Column('id', sa.Integer(), autoincrement=True, primary_key=True),
+        sa.Column('photo',
```

Comment:
There seems to be a problem with this. When I edit just the `description` from the admin page, it deletes the photo.

Reply:
This is a side effect of the TurboGears admin interface when used with depot...

Reply:
@jackrosenthal , hmmm... seems like a pretty bad situation. What if the person editing no longer has access to the original file? Is there literally no way around this? If not, then I wonder if we need to consider a better, more stable solution?

Reply:
Issue is with the way TG admin interface works. Essentially, I think you end up overwriting all columns when updating an entry (there is no interface to just edit certain columns). If you want this fixed, you could always consider forking the TG admin interface and submit a PR, but I don't consider it a big enough deal.

tl;dr, not this project. Submit the issue elsewhere.

Reply:
Fair enough.